FantasmoSDK/app/src/main/java/com/fantasmo/sdk/filters/FMMovementFilter.kt

```diff
@@ -11,18 +10,22 @@ import kotlin.math.abs
  */
 class FMMovementFilter(private val movementFilterThreshold: Float) : FMFrameFilter {
     // Previous frame translation
-    private var lastTransform: FloatArray = floatArrayOf(0F, 0F, 0F)
-
+    private var lastTransform: FloatArray = FloatArray(16){0f}
+    private var lastTimestamp = 0L
```

Comment:
Is `lastTimestamp` is being used elsewhere than just assigning a value in accepts? I believe this is not necessary.